mysql.go

```diff
@@ -87,6 +110,7 @@ type MySQLDatabaseBackup struct {
 }
 
 // MySQLBackupCreateOptions are options used for CreateMySQLDatabaseBackup(...)
+// Deprecated: MySQLBackupCreateOptions is a deprecated struct.
```

Comment:
Can we include some more details about why it was deprecated?

Reply:
Sure, I will add some more details to the deprecation messages I added in this PR.